### PR TITLE
access review policy

### PR DESCRIPTION
**Pull request by kaiwenfeng4086**
Access reviews Policy is a singleton that enable organizations to manage access review directory level policy.

### CODE REVIEW

File: api-reference/beta/resources/accessreviewpolicy.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+doc_type: resourcePageType
+---
+
+# accessReviewPolicy resource type
+
+Namespace: microsoft.graph
+
+[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
+
+Access reviews Policy is a singleton that enable organizations to manage access review directory level policy.
```
Comment (isabelleatmsft) [inline api-reference/beta/resources/accessreviewpolicy.md:16]: This is good, but I think we should add a sentence giving a little more context: "Access review policy is a singleton that enables administrators to manage directory-level access review policies. For example, administrators can use the access review policy to enable and disable the ability of group owners to create access reviews on group that they own."

File: api-reference/beta/resources/accessreviewpolicy.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+## Methods
+|Method|Return type|Description|
+|:---|:---|:---|
+|[Get accessReviewPolicy](../api/accessreviewpolicy-get.md)|[accessReviewPolicy](../resources/accessreviewpolicy.md)|Read the properties and relationships of an [accessReviewPolicy](../resources/accessreviewpolicy.md) object.|
+|[Update accessReviewPolicy](../api/accessreviewpolicy-update.md)|[accessReviewPolicy](../resources/accessreviewpolicy.md)|Update the properties of an [accessReviewPolicy](../resources/accessreviewpolicy.md) object.|
+
+## Properties
+|Property|Type|Description|
+|:---|:---|:---|
+|description|String|Description for this policy.|
```
Comment (isabelleatmsft) [inline api-reference/beta/resources/accessreviewpolicy.md:30]: description and displayName are not editable, correct? If so, please add "Read-only." to the end of the description column for each.

File: api-reference/beta/resources/accessreviewpolicy.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+  "keyProperty": "id",
+  "@odata.type": "microsoft.graph.accessReviewPolicy",
+  "baseType": "microsoft.graph.entity",
+  "openType": false
+}
+-->
+``` json
+{
+  "@odata.type": "#microsoft.graph.accessReviewPolicy",
+  "displayName": "String",
```
Comment (isabelleatmsft) [inline api-reference/beta/resources/accessreviewpolicy.md:50]: The content team usually requests real values filled in here - could you please fill in the values for each property?

**Review by isabelleatmsft — CHANGES_REQUESTED** [review]
Overall looks great. Left a couple comments. Also, could you please update beta/toc.yml to include these new pages in the table of contents?

File: api-reference/beta/resources/accessreviewpolicy.md (hunk 1 above)
Comment (kaiwenfeng4086) [reply api-reference/beta/resources/accessreviewpolicy.md]: updated

File: api-reference/beta/resources/accessreviewpolicy.md (hunk 2 above)
Comment (kaiwenfeng4086) [reply api-reference/beta/resources/accessreviewpolicy.md]: updated

File: api-reference/beta/resources/accessreviewpolicy.md (hunk 3 above)
Comment (kaiwenfeng4086) [reply api-reference/beta/resources/accessreviewpolicy.md]: updated

**Review by isabelleatmsft — APPROVED** [review]

File: api-reference/beta/api/accessreviewpolicy-get.md [hunk 4]
```diff
@@ -0,0 +1,115 @@
+---
+title: "Get accessReviewPolicy"
+description: "Read the properties and relationships of an accessReviewPolicy object."
+author: "kafen"
```
Comment (FaithOmbongi) [inline api-reference/beta/api/accessreviewpolicy-get.md:4]: Global: Please make sure the author is in the [Microsoft Graph contacts list](https://microsoft.sharepoint.com/sites/Graph/Lists/Microsoft%20Graph%20Contacts/AllItems.aspx?CT=1606755556215&OR=OWA%2DNT&CID=798496d0%2D7bc3%2Df8db%2D49ee%2D45ab03388a04&FilterFields1=LinkTitle&FilterValues1=AAD%3B%23AAD%20B2B%3B%23AAD%20B2C%3B%23Core%3B%23Groups&FilterTypes1=Computed&viewid=56fbc209%2Daadb%2D403b%2Da4f7%2Dcb841ae6a70c&id=%2Fsites%2FGraph%2FLists%2FMicrosoft%20Graph%20Contacts). They are the owner of this workload's docs & are responsible for both PRs & GH issues.

File: api-reference/beta/api/accessreviewpolicy-get.md [hunk 5]
```diff
@@ -0,0 +1,115 @@
...
+    "description": "Policy contains directory-level access review settings.",
+    "isGroupOwnerManagementEnabled": false
+  }
+}
+```
+
+### Request
+<!-- {
+  "blockType": "request",
+  "name": "get_accessreviewpolicy"
```
Comment (FaithOmbongi) [inline api-reference/beta/api/accessreviewpolicy-get.md:87]: Global: Differentiate from the first example. For example, "get_accessreviewpolicy_2". Will help when troubleshooting any errors.

File: api-reference/beta/api/accessreviewpolicy-update.md [hunk 6]
```diff
@@ -0,0 +1,111 @@
...
+|Content-Type|application/json. Required.|
+
+## Request body
+In the request body, supply a JSON representation of the [accessReviewPolicy](../resources/accessreviewpolicy.md) object.
+
+The following table shows the properties that are required when you update the [accessReviewPolicy](../resources/accessreviewpolicy.md).
+
+|Property|Type|Description|
+|:---|:---|:---|
+|isGroupOwnerManagementEnabled|Boolean|If true, group owners can create and manage access reviews on groups they own|
```
Comment (FaithOmbongi) [inline api-reference/beta/api/accessreviewpolicy-update.md:50]:
```suggestion
|isGroupOwnerManagementEnabled|Boolean|If `true`, group owners can create and manage access reviews on groups they own.|
```

File: api-reference/beta/api/accessreviewpolicy-update.md [hunk 7]
```diff
@@ -0,0 +1,111 @@
...
+
+
+
+## Response
+
+If successful, this method returns a `204 NoContent` response code.
```
Comment (FaithOmbongi) [inline api-reference/beta/api/accessreviewpolicy-update.md:56]:
```suggestion
If successful, this method returns a `204 No Content` response code.
```

File: api-reference/beta/api/accessreviewpolicy-get.md [hunk 8]
```diff
@@ -0,0 +1,115 @@
...
+[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
+
+Read the properties and relationships of an [accessReviewPolicy](../resources/accessreviewpolicy.md) object.
+
+## Permissions
+One of the following permissions is required to call this API. To learn more, including how to choose permissions, see [Permissions](/graph/permissions-reference).
+
+|Permission type|Permissions (from least to most privileged)|
+|:---|:---|
+|Delegated (work or school account)|Policy.Read.All, Policy.ReadWrite.AccessReviews|
```
Comment (FaithOmbongi) [inline api-reference/beta/api/accessreviewpolicy-get.md:22]: Please ensure **Policy.ReadWrite.AccessReviews** and any other permissions are also documented in the [permissions reference](https://docs.microsoft.com/en-us/graph/permissions-reference).

File: api-reference/beta/api/accessreviewpolicy-update.md [hunk 9]
```diff
@@ -0,0 +1,111 @@
...
+```
+
+### Response
+<!-- {
+  "blockType": "response",
+  "truncated": true
+}
+-->
+``` http
+HTTP/1.1 204 NoContent
```
Comment (FaithOmbongi) [inline api-reference/beta/api/accessreviewpolicy-update.md:83]:
```suggestion
HTTP/1.1 204 No Content
```

File: api-reference/beta/api/accessreviewpolicy-update.md [hunk 10]
```diff
@@ -0,0 +1,111 @@
...
```
Comment (FaithOmbongi) [inline api-reference/beta/api/accessreviewpolicy-update.md:109]:
```suggestion
HTTP/1.1 204 No Content
```

File: api-reference/beta/resources/accessreviewpolicy.md [hunk 11]
```diff
@@ -0,0 +1,55 @@
+---
+title: "accessReviewPolicy resource type"
+description: "Access reviews Policy is a singleton that enable organizations to manage access review directory level policy."
```
Comment (FaithOmbongi) [inline api-reference/beta/resources/accessreviewpolicy.md:3]:
```suggestion
description: "Access reviews policy is a singleton that enables organizations to manage directory-level access review policy."
```

File: api-reference/beta/resources/accessreviewpolicy.md [hunk 12]
```diff
@@ -0,0 +1,55 @@
...
+doc_type: resourcePageType
+---
+
+# accessReviewPolicy resource type
+
+Namespace: microsoft.graph
+
+[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
+
+Access review policy is a singleton that enables administrators to manage directory-level access review policies. For example, administrators can use the access review policy to enable and disable the ability of group owners to create access reviews on group that they own.
```
Comment (FaithOmbongi) [inline api-reference/beta/resources/accessreviewpolicy.md:16]:
```suggestion
Access review policy is a singleton that enables administrators to manage directory-level access review policies. For example, administrators can use the access review policy to enable and disable the ability of group owners to create access reviews on groups that they own.
```

File: api-reference/beta/resources/accessreviewpolicy.md [hunk 13]
```diff
@@ -0,0 +1,55 @@
...
+|:---|:---|:---|
+|[Get accessReviewPolicy](../api/accessreviewpolicy-get.md)|[accessReviewPolicy](../resources/accessreviewpolicy.md)|Read the properties and relationships of an [accessReviewPolicy](../resources/accessreviewpolicy.md) object.|
+|[Update accessReviewPolicy](../api/accessreviewpolicy-update.md)|[accessReviewPolicy](../resources/accessreviewpolicy.md)|Update the properties of an [accessReviewPolicy](../resources/accessreviewpolicy.md) object.|
+
+## Properties
+|Property|Type|Description|
+|:---|:---|:---|
+|description|String|Description for this policy. Read-only.|
+|displayName|String|Display name for this policy. Read-only.|
+|isGroupOwnerManagementEnabled|Boolean|If true, group owners can create and manage access reviews on groups they own|
```
Comment (FaithOmbongi) [inline api-reference/beta/resources/accessreviewpolicy.md:32]:
```suggestion
|isGroupOwnerManagementEnabled|Boolean|If `true`, group owners can create and manage access reviews on groups they own.|
```

File: changelog/Microsoft.IdentityGovernance.AccessReviews.json [hunk 14]
```diff
@@ -81,6 +81,30 @@
...
+            "Target":"policyRoot"
+          },
+          {
+            "Id":"0674e1f3-a14f-11eb-8cb0-33bc249d550e",
+            "ApiChange":"Relationship",
+            "ChangedApiName":"policy",
+            "ChangeType":"Addition",
+            "Description":"Added the **policy** relationship to the [accessReviewSet](https://docs.microsoft.com/en-us/graph/api/resources/accessReviewSet?view=graph-rest-beta) resource.",
+            "Target":"accessReviewSet"
+          },
```
Comment (FaithOmbongi) [inline changelog/Microsoft.IdentityGovernance.AccessReviews.json:100]: Can't see **policyRoot**, **accessReviewSet** resources in the public doc set. Are you introducing them in another PR? If not, kindly remove these references.

File: changelog/Microsoft.IdentityGovernance.AccessReviews.json [hunk 15]
```diff
@@ -81,6 +81,30 @@
...
+          {
+            "Id":"067741e8-a14f-11eb-8cb0-33bc249d550e",
+            "ApiChange":"Resource",
+            "ChangedApiName":"accessReviewPolicy",
+            "ChangeType":"Addition",
+            "Description":"Added the [accessReviewPolicy](https://docs.microsoft.com/en-us/graph/api/resources/accessReviewPolicy?view=graph-rest-beta) resource type.",
+            "Target":"accessReviewPolicy"
           }
```
Comment (FaithOmbongi) [inline changelog/Microsoft.IdentityGovernance.AccessReviews.json:108]: Kindly add this PR's changelog entry as a separate object, first entry; not within an existing object. As you do that, please modify the value of access reviews SubArea to **Governance**.

**Review by FaithOmbongi — CHANGES_REQUESTED** [review]

File: api-reference/beta/api/accessreviewpolicy-get.md (hunk 4 above)
Comment (isabelleatmsft) [reply api-reference/beta/api/accessreviewpolicy-get.md:4]: @kaiwenfeng4086 you can put "isabelleatmsft" as the author here so we dont have to expand the contacts list too much

File: api-reference/beta/api/accessreviewpolicy-update.md [hunk 16]
```diff
@@ -0,0 +1,111 @@
...
+### Request
+<!-- {
+  "blockType": "request",
+  "name": "update_accessreviewpolicy"
+}
+-->
+``` http
+PATCH https://graph.microsoft.com/beta/policies/accessReviewPolicy
+Content-Type: application/json
+Content-length: 163
```
Comment (davidmu1) [inline api-reference/beta/api/accessreviewpolicy-update.md:69]: Content-length isn't needed in the examples.

**Review by davidmu1 — APPROVED** [review]
Agree with Faith's comments.

File: api-reference/beta/api/accessreviewpolicy-get.md (hunk 8 above)
Comment (isabelleatmsft) [reply api-reference/beta/api/accessreviewpolicy-get.md:22]: did this in this PR: https://github.com/microsoftgraph/microsoft-graph-docs/pull/12506

**Review by isabelleatmsft — APPROVED** [review]

**Review by FaithOmbongi — APPROVED** [review]

**Pull request merged**